test/test_maskedtensor.py

```diff
@@ -281,84 +439,84 @@ def test_mean(self):
     """
         The following block of tests "test_mean_grad_case_1[a through e] are used to test the functionality of
         the two different ways of constructing MaskedTensors:
-            MaskedTensor(data, mask, requires_grad=True/False) -- NO differentiable constructor and always a leaf
-            MaskedTensor.from_values(data, mask) -- differentiable constructor
+            masked_tensor(data, mask, requires_grad=True/False) -- NO differentiable constructor and always a leaf
+            masked_tensor(data, mask) -- differentiable constructor
```

Comment:
Did you mean `as_masked_tensor` as in the piece of text below? Please check for this error within the PR and ask for review once you're confident it's been resolved.

Reply:
Yeah thanks, fixed